00 Intro/03 Output/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "boilerplate",
+  "name": "output",
   "version": "1.0.0",
-  "description": "Front-End Lemoncode Master, Bundle Modules, Webpack Demo 00 Boilerplate",
+  "description": "Front-End Lemoncode Master, Bundle Modules, Webpack Demo 03 Output",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server -p"
```

Comment:
We don't should to assign `-p` to webpack-dev-server because it's a server for development and `-p` y for production and it will used by webpack